browser/pages/account/controller.js

```diff
@@ -83,6 +93,12 @@ class AccountController {
       this.isLoginBtnClicked = false;
       this.router.go('install');
       this.authFailed = false;
+      // Storing the password for next use
+      let dataFilePath = path.join(remote.app.getPath('userData'),'RedHat','DevelopmentSuite');
+      mkdirp.sync(dataFilePath);
+      let data = {'username': this.username};
+      fs.writeFileSync(path.join(dataFilePath, 'settings.json'), JSON.stringify(data));
+      this.scope.credential.TokenStore.setItem('login', this.username, this.password);
```

Comment:
This call should be done only if `[ ] Remember me` checkbox under username/login is selected.